### PR TITLE
[cadence-cli] fix typo in input flag for parallelism

### PR DESCRIPTION
**Pull request by sankari165**
fix typo in input flag  for parallelism

### CODE REVIEW

**Review by jakobht — APPROVED** [review]

**Comment by dkrotx** [comment]
We can use this trick to avoid changing plenty of places this is already used (including OSS cases we not aware of).
https://go.dev/play/p/1fij2v-1cO_l

File: tools/cli/workflowCommands.go [hunk 1]
```diff
@@ -1510,6 +1510,9 @@ func ResetInBatch(c *cli.Context) {
 	excludeQuery := c.String(FlagExcludeWorkflowIDByQuery)
 	separator := c.String(FlagInputSeparator)
 	parallel := c.Int(FlagParallism)
```
Comment (dkrotx) [inline tools/cli/workflowCommands.go:1512]: it's default-value is 1, not 0.

File: tools/cli/flags.go [hunk 2]
```diff
@@ -93,6 +93,7 @@ const (
 	FlagSignalInputFileWithAlias          = FlagSignalInputFile + ", sif"
 	FlagExcludeFile                       = "exclude_file"
 	FlagInputSeparator                    = "input_separator"
+	FlagParallelism                       = "input_parallelism"
```
Comment (dkrotx) [inline tools/cli/flags.go:96]:
```suggestion
	FlagParallelismDeprecated                       = "input_parallelism" // typo, replaced by FlagParallism
```

File: tools/cli/workflowCommands.go [hunk 3]
```diff
@@ -1510,6 +1510,9 @@ func ResetInBatch(c *cli.Context) {
...
+	if parallel == 0 {
```
Comment (dkrotx) [inline tools/cli/workflowCommands.go:1513]: it's default-value is 1, not 0.

File: tools/cli/flags.go (hunk 2 above)
Comment (sankari165) [reply tools/cli/flags.go:96]: updated

**Review by dkrotx — APPROVED** [review]

File: tools/cli/workflowCommands.go (hunk 3 above)
Comment (sankari165) [reply tools/cli/workflowCommands.go]: updated

**Pull request merged**